azerothcore/azerothcore-wotlk
# fix(DB/formations): AQ40 Stationary pack linking

## Changes Proposed:
-  Adds formations to stationary/wandering packs in AQ40
- Links Anibisath Defenders to Twin Emperors so that they are pulled if not killed beforehand (source is retail)
- Edit: And everything after Twin Emperors is not touched on since they're all broken, missing patrol formations and waypoints.

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/3808
- Closes https://github.com/chromiecraft/chromiecraft/issues/3806

## SOURCE:


## Tests Performed:

- Tested ingame


## How to Test the Changes:


1. Brainwasher Pack `.go xyz -8428.73 2047.70 116.1214 531 0.44`
2. Brainwasher Pack `.go xyz -8565 1943 2.68527 531 2.3`
3. Two Brainwasher & Guardian packs `.go xyz -8591.233 2074.435303 -1.465834 531 1.544`
4. Vekniss Guardian pack `.go xyz -8432.95 2034.7 8.914 531 0.396`
5. Vekniss Guardian Tunnel, 3 packs `.go xyz -8406.5 1999.4 7.9 531 5`
6. Anubisath Defender tunnel `.go xyz -8958 1263 -112.3 531 4.9` Pull one of the emperors, see if the Defenders enter combat. If despawned due to distance, Anubisath Defenders might not engage in combat with the player
7. Anubisath Warder & Obsidian Nullifier 4 packs `.go xyz -9160 1732 -41.186 531 0.865`

## Known Issues and TODO List:


- [ ] Vekniss Warriors, Hive Crawlers, Wasp packs and patrolling trash after Twin Emperors.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
